Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -456,7 +456,7 @@ public async Task RequestHandler(IHttpRequest httpReq, string urlString, string
             var stopWatch = new Stopwatch();
             stopWatch.Start();
             var httpRes = httpReq.Response;
-            string urlToLog = null;
+            string urlToLog = GetUrlToLog(urlString);
```

Comment:
Would be better to call this in the `catch { }` blocks since it's not used anywhere else before then, but this is fine

Reply:
It's used in `finally` which might trigger without `catch{}`, that's the reason I put it there.

Reply:
Ah yes, you're right. I read that wrong.